source/socks5-support/socks5.rst

```diff
@@ -0,0 +1,186 @@
+==============
+SOCKS5 Support
+==============
+
+:Spec Title: SOCKS5 Support
+:Spec Version: 1.0.0
+:Author: Anna Henningsen
+:Advisors: TODO
+:Status: Draft
+:Type: Standards
+:Minimum Server Version: N/A
+:Last Modified: 2021-11-19
+
+.. contents::
+
+--------
+
+Abstract
+========
+
+This specification defines driver behaviour when connecting to MongoDB services
+through a SOCKS5 proxy.
+
+META
+====
+
+The keywords "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD",
+"SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be
+interpreted as described in `RFC 2119 <https://www.ietf.org/rfc/rfc2119.txt>`__.
+
+Specification
+=============
+
+
+Terms
+-----
+
+SOCKS5
+^^^^^^
+
+The SOCKS protocol, version 5, as defined in `RFC1928 <https://datatracker.ietf.org/doc/html/rfc1928>`__,
+restricted to either no authentication or username/password authentication
+as defined in `RFC1929 <https://datatracker.ietf.org/doc/html/rfc1929>`__.
+
+
+MongoClient Configuration
+-------------------------
+
```

Comment:
I would add these new options to the table at https://github.com/mongodb/specifications/blob/master/source/uri-options/uri-options.rst#list-of-specified-options as well, and put a section in that spec just above for SOCKS5 options linking back to this spec.

Reply:
Thanks for mentioning this, done!